app/src/main/java/com/andrew/tasky/agenda/domain/models/AgendaItem.kt

```diff
@@ -18,7 +19,7 @@ sealed interface AgendaItem : Serializable {
         val isAttendee: Boolean,
         val attendees: List<Attendee>? = emptyList(),
         val host: String
-    ) : AgendaItem
+    ) : AgendaItem, Serializable
```

Comment:
I'd prefer `Parcelable` over `Serializable` which is just faster. You can use the Gradle Parcelize plugin to easily create parcelables simply by adding the `@Parcelize` annotation